plugins/reach/admin/scripts/plugin/partner-catalog-item-list-action.phtml

```diff
@@ -59,8 +59,16 @@
 				else
 				    echo null;?>
 				</td>
-				<td><?php if (isset($catalogItem->outputFormat))
-				    echo $this->enumTranslate('Kaltura_Client_Reach_Enum_VendorCatalogItemOutputFormat', $catalogItem->outputFormat);
+				<td><?php if (isset($catalogItem->outputFormat)) {
```

Comment:
Why isn't VendorCatalogItemSignLanguageOutputFormat an extension of VendorCatalogItemOutputFormat
This would remove the need if this if and will make much more sense

Reply:
But the values for both VendorCatalogItemSignLanguageOutputFormat and VendorCatalogItemOutputFormat are the same, the only difference is the view in the KAC
![image](https://github.com/user-attachments/assets/8ce3a05e-6a2d-47ea-983a-2510cd0e4e6c)
For example in catalog item list:
![image](https://github.com/user-attachments/assets/b0a5ad4d-4a77-40d4-be12-aec12113df28)

Reply:
This implements base enum
<img width="1122" alt="image" src="https://github.com/user-attachments/assets/cb1b2072-45d4-401b-8700-3d666f592236" />

Whereas it could extend this:
<img width="667" alt="image" src="https://github.com/user-attachments/assets/bf06c832-258f-4d4f-95ee-ccf69a34c2c6" />

Reply:
@yossipapi  I think it's better to eliminate KalturaVendorSignLanguageFormat, and add the 2 new values to the base enum, since this is part of the base reach capability anyway. Keeping the enum and making it extend the base class risks overlap of enum values in the future.